src/test/ui/const-generics/issues/issue-66205.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(incomplete_features, dead_code, unconditional_recursion)]
 #![feature(const_generics)]
+#![feature(lazy_normalization_consts)]
```

Comment:
I'm not sure we want to be adding this to every `const_generics` test. We should instead make `#![feature(const_generics)]` subsume `#![feature(lazy_normalization_consts)]`.